### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by alexander-daniel**
@etpinard 

Fixed!

### CODE REVIEW

File: plotly-raspi-stream.py [hunk 1]
```diff
@@ -1,24 +1,14 @@
...
-stream_token = plotly_user_config['plotly_streaming_tokens'][0]
-stream_server = 'http://stream.plot.ly'
+username = 'your_plotly_username'
+api_key = 'your_api_key'
+stream_token = 'your_stream_token'
 
 py.sign_in(username, api_key)
 
-print p.plot([{'x': [], 'y': [], 'type': 'scatter',
+print py.plot([{'x': [], 'y': [], 'type': 'scatter',
```
Comment (etpinard) [inline plotly-raspi-stream.py]: Maybe you could convert that to a `Scatter` graph object as in README.md ?

File: README.md [hunk 2]
```diff
@@ -14,20 +14,6 @@ sudo pip install rpi.gpio
...
-{
-"plotly_streaming_tokens": ["your_stream_token", "another_stream_token"],
-"plotly_api_key": "your_api_key",
-"plotly_username": "your_user_name"
-}
-```
-
 Create your sensor reading script, and start importing some modules in it!
 ```python
 import plotly.plotly as py # plotly library
```
Comment (etpinard) [inline README.md:19]: maybe have `from plotly.graph_objs import *` instead, as in the [docs](https://plot.ly/python/bubble-charts/)

**Comment by alexander-daniel** [comment]
Sweet! Yeah sorry, I'm so used to the old syntax. Im on it!

**Comment by etpinard** [comment]
:+1: :+1: :+1: 

Merge worthy for sure.

**Pull request merged**